package.json

```diff
@@ -41,6 +46,6 @@
     "publish-please": "^5.1.1",
     "tap-colorize": "^1.2.0",
     "tape": "^4.9.1",
-    "typescript": "^2.6.1"
+    "typescript": "^2.8.1"
```

Comment:
May as well bump to 2.8.3, [which includes some bug fixes](https://github.com/microsoft/TypeScript/releases/tag/v2.8.3)